client/app/validations/saveable-landuse-form.js

```diff
@@ -0,0 +1,141 @@
+import {
+    validateFormat,
+    validateLength,
+  } from 'ember-changeset-validations/validators';
+  
+  // These validate the fields for _saving_ to the server
+export default {
+  dcpContactname: [
+    validateLength({
+      min: 0,
+      max: 100,
+      message: 'Text is too long (max {max} characters)',
+    }),
+  ],
+  dcpSitedataadress: [
+    validateLength({
+      min: 0,
+      max: 250,
+      message: 'Text is too long (max {max} characters)',
+    }),
+  ],
+  dcpContactemail: [
+    validateLength({
+      min: 0,
+      max: 100,
+      message: 'Text is too long (max {max} characters)',
+    }),
+    validateFormat({
+      type: 'email',
+      // Set allowBlank=true so that the validation message
+      // only appears after user first types sometihng.
+      // This field still indicates it is 'required'
+      // through validatePresence within
+      // submittable-applicant-form
+      allowBlank: true,
+      message: 'Must be a valid email address',
+    }),
+  ],
+  dcpContactphone: [
+      validateLength({
+        min: 0,
+        max: 10,
+        message: 'Text is too long (max {max} characters)',
+      }),
+      validateFormat({
+        type: 'phone',
+        // Set allowBlank=true so that the validation message
+        // only appears after user first types sometihng.
+        // This field still indicates it is 'required'
+        // through validatePresence within
+        // submittable-applicant-form
```

Comment:
Can we remove or update these comments here?

Reply:
Yep!